elife/newrelic.sls

```diff
@@ -1,61 +1,78 @@
+# newrelic 'servers' (newrelic.sls) is now deprecated in favour of 
```

Comment:
would split this in two parts: 1. change `newrelic-infrastructure` and 2. change `newrelic`; so that we can turn on selectively on some hosts `newrelic-infrastructure` via `top.sls` and iron it out before pushing it to everyone; all the time while continuing to run `newrelic` to keep monitoring the servers until the replacement is ready